gateway/1.7/modules/ROOT/pages/flex-troubleshoot-linux-permissions.adoc

```diff
@@ -0,0 +1,53 @@
+= Troubleshooting Linux Permissions
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+In Linux, to limit access to information about a Flex Gateway, the following `flexctl` commands require that user belong to the `flex` user group or to be the root user to execute them:
```

Comment:
This sentence is a bit unwieldy. The lead in doesn't quite seem to apply. What you're actually talking about is the permissions needed for the specific commands, right? So how about something like:
The following `flexctl` commands can be executed on Linux only by the root user and by users in the `flex` user group. This helps limit access to information about a Flex Gateway.

Reply:
This is an example of where trying to make a sentence in active voice can result in a less clear sentence. Passive voice is okay when it makes sense, and it often does for things like permissions and command line commands.